privacy-config/privacy-config-api/src/main/java/com/duckduckgo/privacy/config/api/PrivacyConfig.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.config.api
+
+import com.duckduckgo.feature.toggles.api.FeatureName
+
+interface PrivacyConfigDownloader {
+    suspend fun download(): Boolean
+}
```

Comment:
This seems to only be used by the `privacy-config-impl`. 
Do you think the `download` method will be necessary for any other module that it's not `privacy-config-impl`. If only the `impl` is going to use it, maybe we can move this interface to the `impl` for now?

Reply:
It made sense when download was managed by the features module but not anymore. Moved to `impl`!